### PR TITLE
Add PRESENCE token metadata

### PR DESCRIPTION
**Pull request by bridgitbrowser**
Add PRESENCE token metadata
ERC-20: https://etherscan.io/address/0xC6903B623f1548f533Eb367f6f1B7d717b9351c2
NEP-141: https://explorer.mainnet.near.org/accounts/c6903b623f1548f533eb367f6f1b7d717b9351c2.factory.bridge.near
Aurora ERC-20 (BlockScout): https://explorer.mainnet.aurora.dev/address/0xf6db8e249099b909a764e46a9cc0c9309dcfd0ce
Aurora ERC-20 (Aurorascan): https://aurorascan.dev/address/0xf6db8e249099b909a764e46a9cc0c9309dcfd0ce

### CODE REVIEW

**Comment by bridgitbrowser** [comment]
I just added the Aurora token

**Comment by bridgitbrowser** [comment]
Hi there, it put these two tokens together in the same pull request. I hope that's ok.

**Comment by sept-en** [comment]
@bridgitbrowser please split two tokens into separate PRs. This is needed for sanity.

**Comment by bridgitbrowser** [comment]
Ok, I removed the BRGT so the pull request is just the PRESENCE token

**Comment by bridgitbrowser** [comment]
I this ok? the pull request is just PRESENCE now

File: tokens/presence.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0xC6903B623f1548f533Eb367f6f1B7d717b9351c2",
+    "aurora_address": "0xf6db8e249099b909a764e46a9cc0c9309dcfd0ce",
+    "name": "Pressence DAO",
```
Comment (sept-en) [inline tokens/presence.json:4]:
```suggestion
    "name": "Presence DAO",
```

**Review by sept-en — CHANGES_REQUESTED** [review]

**Review by sept-en — APPROVED** [review]

**Pull request merged**